__tests__/Itinerary.test.js

```diff
@@ -0,0 +1,15 @@
+const Itinerary = require('../src/Itinerary.js');
+
+describe('Itinerary', () => {
+    const ramkinHallPort = {name: "Ramkin Hall", ships: [], addShip: jest.fn(), removeShip: jest.fn()};
+    const quirmPort = {name: "Quirm", ships: [], addShip: jest.fn(), removeShip: jest.fn()};
+    const itineraryOne = new Itinerary([ramkinHallPort, quirmPort]);
+    
+    it('can be instantiated', () => {
+        expect(itineraryOne).toBeInstanceOf(Object);
```

Comment:
You can be a bit more specific here and say exactly what type of object you are expecting:
```suggestion
        expect(itineraryOne).toBeInstanceOf(Itinerary);
```